src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.spec.ts

```diff
@@ -11,77 +11,65 @@ import { TestBed } from '@angular/core/testing';
 describe('DotEditLayoutService', () => {
     const containers = {
```

Comment:
can you put all containers data into a single file to be reused?
src/app/portlets/dot-edit-page/template-containers-cache.spec.ts